code/src/components/TaskList.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { useSelector } from 'react-redux'
+import 'components/TaskList.css'
+import { Task } from 'components/Task'
+
+
+export const TaskList = () => {
+
+    const allTasks = useSelector((state) => state.tasks.items)
+    console.log(allTasks)
```

Comment:
If you want to order the uncompleted and completed you could have them in two arrays.
Like this:
`const completedTasks = useSelector((state) => state.tasks.items.filter((item) => item.needsMore)`
`const uncompletedTasks = useSelector((state) => state.tasks.items.filter((item) => !item.needsMore)`